draft-ietf-pquip-pqc-hsm-constrained.md

```diff
@@ -94,47 +100,64 @@ informative:
-and performance optimizations specific to resource-limited environments. One approach to
-mitigating storage constraints is seed-based key generation, where only a small seed is
-stored instead of the full private key, as supported by PQC schemes like ML-DSA and
-SLH-DSA. However, this technique increases computational overhead due to the need to derive
-full private keys on demand,  a classic computation-versus-storage tradeoff. The document
-also discusses considerations for ephemeral key generation in protocols like TLS and
-IPsec, along with techniques to optimize PQC signature operations to enhance performance
-within constrained crytographic modules.
-
-This document focuses on post‑quantum cryptography in constrained devices, specifically on the three algorithms finalized by NIST: ML-DSA, ML-KEM, and SLH-DSA and on stateful hash‑based signatures in the context of firmware signing. Future revisions may expand the scope to include additional PQC algorithms.
+The transition to post-quantum cryptography (PQC) poses significant challenges for
+resource-constrained Internet of Things (IoT) devices, which are often equipped with
+Trusted Execution Environments (TEEs), secure elements, or other forms of hardware
+security modules (HSMs). These devices typically operate under strict limitations on
+processing power, RAM, and flash memory, and in some cases are battery-powered. Adopting
+PQC algorithms in such environments is difficult due to their substantially larger key
+sizes and, in some cases, higher computational demands. Consequently, the migration to
+PQC requires careful planning to ensure secure and efficient key management within
+constrained platforms.
+
+This document provides guidance and best practices for integrating PQC algorithms into
+constrained devices. It reviews strategies for key storage, ephemeral key management,
+and performance optimization tailored to low-resource environments. One mitigation
+technique for storage limitations is seed-based key generation, where only a compact
+seed is stored instead of the full private key. While this approach conserves memory,
+it introduces additional computational overhead because full private keys must be derived
+on demand—illustrating the classic computation-versus-storage trade-off. The document also
+examines ephemeral key generation in protocols such as TLS, along with techniques to
+optimize PQC signature operations to improve performance within constrained cryptographic
+modules.
+
+The focus is on PQC in constrained devices, with particular attention to the three
+algorithms standardized by NIST:
+
+- Module-Lattice-Based Key-Encapsulation Mechanism (ML-KEM) {{ML-KEM}},
+- Module-Lattice-Based Digital Signature Algorithm (ML-DSA) {{ML-DSA}}, and
+- Stateless Hash-Based Digital Signature Algorithm (SLH-DSA) {{SLH-DSA}}.
+
+The Hierarchical Signature System (HSS) / Leighton–Micali Signature (LMS) {{RFC8554}} is
```

Comment:
Should be ``The Hierarchical Signature System/Leighton–Micali Signature (HSS/LMS)`` (we refer to HSS/LMS in the document).